modulefiles/gaeac6.lua

```diff
@@ -0,0 +1,28 @@
+help([[
+  This module loads libraries required for building and running UPP
+  on the NOAA RDHPC machine Gaea C6 using Intel-2023.2.0.
+]])
+
+whatis([===[Loads libraries needed for building the UPP on Gaea C6 ]===])
+
+prepend_path("MODULEPATH", "/autofs/ncrc-svm1_proj/epic/spack-stack/spack-stack-1.6.0/envs/upp-addon-env/install/modulefiles/Core")
+
+stack_intel_ver=os.getenv("stack_intel_ver") or "2023.2.0"
+load(pathJoin("stack-intel", stack_intel_ver))
+
+stack_cray_mpich_ver=os.getenv("stack_cray_mpich_ver") or "8.1.28"
```

Comment:
@DavidBurrows-NCO The UFSWM PR #[2448](https://github.com/ufs-community/ufs-weather-model/pull/2448) uses 8.1.29. Would you like to keep the version consistent with UFSWM?

Reply:
@WenMeng-NOAA I checked into using `stack-cray-mpich.v8.1.29` on Gaea-C6.  To do that, I would have to switch `spack-stack` environment to `unified-env-c6`, however, `unified-env-c6` only contains `g2.v3.4.5` and `g2tmpl.v1.10.2`.

Reply:
@DavidBurrows-NCO Thanks for checking and explanation. In that case, we keep the current settings for building offline post.